UI/window-extra-browsers.cpp

```diff
@@ -234,11 +233,13 @@ void ExtraBrowsersModel::Apply()
 
 	for (int i = deleted.size() - 1; i >= 0; i--) {
 		int idx = deleted[i];
-		main->extraBrowserDockActions.removeAt(idx);
 		main->extraBrowserDockTargets.removeAt(idx);
 		main->extraBrowserDocks.removeAt(idx);
 	}
 
+	if (main->extraBrowserDocks.empty())
+		main->extraBrowserMenuDocksSeparator.clear();
+
```

Comment:
If the user doesn't initially have browser docks, I'm not seeing a place where this gets inserted when the user creates their first custom browser dock.

Reply:
Here it the the new `AddDockWidget` function:

https://github.com/obsproject/obs-studio/pull/7636/files#diff-4c0a52244f264feca9c3031c90edcbbbd1087a3040c15f9df4ca9700c5154fa3R9910-R9922

Reply:
Ah, you're right, I'm just blind.

Reply:
I need to add the check for separator to the old one too, to add the dock before the separator 🤦.